substrate/frame/executive/src/lib.rs

```diff
@@ -188,6 +188,7 @@ impl<
 		Context: Default,
 		UnsignedValidator,
 		AllPalletsWithSystem: OnRuntimeUpgrade
+			+ BeforeAllRuntimeMigrations
```

Comment:
With the direction that MBM+HDH Pr is taking, I am looking forward to have less things that are configured in executive:)